samples/deleteSnippets.js

```diff
@@ -0,0 +1,153 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  instanceId = 'YOUR_INSTANCE_ID',
+  tableId = 'YOUR_TABLE_ID',
+  deleteType = 'readRow'
+) {
+  // [START bigtable_deletes_print]
+  const {Bigtable} = require('@google-cloud/bigtable');
+  const bigtable = new Bigtable();
+
+  // TODO: We need to import Mutation.methods.DELETE from mutation.ts, but I guess we need to compile typescript?
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const instanceId = 'YOUR_INSTANCE_ID';
+  // const tableId = 'YOUR_TABLE_ID';
+  const instance = bigtable.instance(instanceId);
+  const table = instance.table(tableId);
+
+  switch (deleteType) {
+    case 'deleteFromColumn': {
+      // [START bigtable_delete_from_column]
+      await table.mutate({
+        key: 'phone#4c410523#20190501',
+        method: 'delete',
+        data: {
+          column: 'cell_plan:data_plan_05gb',
+        },
+      });
+      await printRows();
+      // [END bigtable_delete_from_column]
+      break;
+    }
+    case 'deleteFromFamily': {
+      // [START bigtable_delete_from_column_family]
+      await table.mutate({
+        key: 'phone#4c410523#20190501',
+        method: 'delete',
+        data: {
+          column: 'cell_plan',
+        },
+      });
+      await printRows();
+      // [END bigtable_delete_from_column_family]
+      break;
+    }
+    case 'deleteFromRow': {
+      // [START bigtable_deletes_from_row]
+      const row = table.row('phone#4c410523#20190501');
+      await row.deleteCells(['stats_summary:connected_cell']);
+      await printRows();
+      // [END bigtable_deletes_from_row]
+      break;
+    }
+    case 'streamingAndBatching': {
+      // [START bigtable_streaming_and_batching]
+      const rows = (await table.getRows({limit: 1}))[0];
+      const entries = rows.map(row => {
+        return {
+          key: row.id,
+          method: 'delete',
+        };
+      });
+      await table.mutate(entries);
+      await printRows();
+      // [END bigtable_streaming_and_batching]
+      break;
+    }
+    case 'checkAndMutate': {
+      // [START bigtable_check_and_mutate]
+      const row = table.row('phone#4c410523#20190502');
+      await row.filter(
+        {
+          column: 'data_plan_05gb',
+        },
+        {
+          onMatch: {
+            key: 'phone#4c410523#20190502',
+            method: 'delete',
+          },
+        }
+      );
+      await printRows();
+      // [END bigtable_check_and_mutate]
+      break;
+    }
+    case 'dropRowRange': {
+      // [START bigtable_drop_row_range]
+      await table.deleteRows('phone#5c10102#201');
```

Comment:
We usually describe row key prefixes as being delimited by separators like the hash tag, so I think it might reinforce that idea if the row key prefix used here is just phone#5c10102

Reply:
Sounds good to me.